### ISSUE TITLE
ifm.php missing in Docker build

### ISSUE THREAD

**Issue by lbesnard** [misterunknown/ifm#135]
Trying to build ifm with docker, unfortunately this fails towards the last few steps with a missing ifm.php which I can't find in the repo

```
Step 8/12 : COPY ifm.php /usr/local/share/webapps/ifm/index.php
ERROR: Service 'ifm' failed to build: COPY failed: stat /var/lib/docker/tmp/docker-builder*****/ifm.php: no such file or directory
```

**Comment by misterunknown** [comment]
Indeed. I fix that.

**Comment by misterunknown** [comment]
Hi, this should be fixed in the latest release v2.6.2. Please let me know if it still does not work for you.

**Comment by lbesnard** [comment]
all good thanks!
